### ISSUE TITLE
Pseudo-terminal will not be allocated

### ISSUE THREAD

**Issue by exeral** [rprichard/wslbridge#45]
Hello,

I am using wslbridge with ConEmu (191012)

since few days, I have this error
_Pseudo-terminal will not be allocated because stdin is not a terminal._
with a weird shell

My ConeEmu task is
`set "PATH=%ConEmuBaseDirShort%\wsl;%PATH%" & %ConEmuBaseDirShort%\conemu-cyg-64.exe --wsl -cur_console:pm:/mnt -C "~" -t zsh`

a ConEmu tasks without `conemu-cyg-64.exe --wsl` works properly

**Comment by mavaddat** [comment]
For recent changes in WSL, particularly WSL mode V2, the new gateway [wslbridge2](https://github.com/Biswa96/wslbridge2) should be used instead. Many thanks for this further development and maintenance go to [Biswapriyo Nath](https://github.com/Biswa96).

**Comment by exeral** [comment]
Yes, I heard of that. But for now, it seems there is no doc on how to use it with ConEmu
